bin/v-change-web-domain-docroot

```diff
@@ -80,21 +80,28 @@ if [ "$target_domain" = "default" ]; then
     update_object_value 'web' 'DOMAIN' "$domain" '$CUSTOM_PHPROOT' "" 
 else
     # Check for existence of specified directory under target domain's public_html folder
+    target_domain_directory="$HOMEDIR/$user/web/$target_domain/"
```

Comment:
It adds an additional slash between the domain and public_html folder 
Edit domain1.tld and pointed to domain2.tld and folder "jaap"

CUSTOM_PHPROOT='/home/demo/webdomain2.tld//public_html/' 
CUSTOM_DOCROOT='/home/demo/web/domain2.tld/public_html/jaap'

Folder empty:
CUSTOM_PHPROOT='/home/demo/web/domain2.tld//public_html/' CUSTOM_DOCROOT='/home/demo/web/domain2.tld/public_html/'

Docroot set to domain 1:
CUSTOM_PHPROOT='/home/demo/web/domain1.com//public_html/'   CUSTOM_DOCROOT='/home/demo/web/domain1.com/public_html/jaap'

Also added between 90 - 104